localstripe/resources.py

```diff
@@ -4018,7 +4024,9 @@ class IssuingDispute(StripeObject):
     _id_length = 24
 
     def __init__(self, transaction: str, currency: str = 'usd', metadata: dict = None, evidence: dict = None, amount = None):
-        amount = try_convert_to_int(amount)
+        if amount is not None:
```

Comment:
I don't think you need to have this though...?

Reply:
removed